.golangci.next.reference.yml

```diff
@@ -2814,6 +2814,12 @@ issues:
   # Default: false
   exclude-case-sensitive: false
 
+  # To follow strict Go autogenerated file convention.
+  # https://go.dev/s/generatedcode
+  # By default a lax pattern is applied.
+  # Default: false
+  exclude-autogenerated-strict: true
```

Comment:
Should we add a new option that allows linting of the generated files?
```
# If exclude generated files.
# Default: true
exclude-generated: false
```
Of course, this is a separate feature.